testutils/g103_samples.go

```diff
@@ -27,7 +27,7 @@ func main() {
 	intPtr = (*int)(unsafe.Pointer(addressHolder))
 	fmt.Printf("\nintPtr=%p, *intPtr=%d.\n\n", intPtr, *intPtr)
 }
-`}, 3, gosec.NewConfig()},
+`}, 2, gosec.NewConfig()},
```

Comment:
Why did these need to be changed? The logic of the analyzer/rule didn't change.

Reply:
The 3 here represent two - G103 Rules and one - G115(analyzer - Type conversion which leads to integer overflow).
With the analyzers no longer being included. G115 is now excluded and hence the correct value is "2".